packages/contracts/slither.db.json

```diff
@@ -0,0 +1 @@
, 68], "starting_column": 1, "ending_column": 4}}, "signature": "appendL2Output(bytes32,uint256)"}}, {"type": "node", "name": "require(bool,string)(_timestamp == nextTimestamp(),Timestamp not equal to next expected timestamp)", "source_mapping": {"start": 2010, "length": 88, "filename_used": "/Users/maurelian/Projects/O/optimistic-specs/packages/contracts/contracts/L1/MockL2OutputOracle.sol", "filename_relative": "contracts/L1/MockL2OutputOracle.sol", "filename_absolute": "/Users/maurelian/Projects/O/optimistic-specs/packages/contracts/contracts/L1/MockL2OutputOracle.sol", "filename_short": "contracts/L1/MockL2OutputOracle.sol", "is_dependency": false, "lines": [48], "starting_column": 9, "ending_column": 97}, "type_specific_fields": {"parent": {"type": "function", "name": "appendL2Output", "source_mapping": {"start": 1769, "length": 422, "filename_used": "/Users/maurelian/Projects/O/optimistic-specs/packages/contracts/contracts/L1/MockL2OutputOracle.sol", "filename_relative": "contracts/L1/MockL2OutputOracle.sol", "filename_absolute": "/Users/maurelian/Projects/O/optimistic-specs/packages/contracts/contracts/L1/MockL2OutputOracle.sol", "filename_short": "contracts/L1/MockL2OutputOracle.sol", "is_dependency": false, "lines": [45, 46, 47, 48, 49, 50, 51], "starting_column": 5, "ending_column": 6}, "type_specific_fields": {"parent": {"type": "contract", "name": "MockL2OutputOracle", "source_mapping": {"start": 95, "length": 2771, "filename_used": "/Users/maurelian/Projects/O/optimistic-specs/packages/contracts/contracts/L1/MockL2OutputOracle.sol", "filename_relative": "contracts/L1/MockL2OutputOracle.sol", "filename_absolute": "/Users/maurelian/Projects/O/optimistic-specs/packages/contracts/contracts/L1/MockL2OutputOracle.sol", "filename_short": "contracts/L1/MockL2OutputOracle.sol", "is_dependency": false, "lines": [7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68], "starting_column": 1, "ending_column": 4}}, "signature": "appendL2Output(bytes32,uint256)"}}}}], "description": "MockL2OutputOracle.appendL2Output(bytes32,uint256) (contracts/L1/MockL2OutputOracle.sol#45-51) uses a dangerous strict equality:\n\t- require(bool,string)(_timestamp == nextTimestamp(),Timestamp not equal to next expected timestamp) (contracts/L1/MockL2OutputOracle.sol#48)\n", "markdown": "[MockL2OutputOracle.appendL2Output(bytes32,uint256)](contracts/L1/MockL2OutputOracle.sol#L45-L51) uses a dangerous strict equality:\n\t- [require(bool,string)(_timestamp == nextTimestamp(),Timestamp not equal to next expected timestamp)](contracts/L1/MockL2OutputOracle.sol#L48)\n", "first_markdown_element": "contracts/L1/MockL2OutputOracle.sol#L45-L51", "id": "74ee9955ede80e105221045625f8b4f963a0ca8422674891166ad65bdcdba57a", "check": "incorrect-equality", "impact": "Medium", "confidence": "High"}]
```

Comment:
hardcoded paths (referencing your machine) are stored in this file

Reply:
ewwww